sqlg-core/src/main/java/org/umlg/sqlg/structure/SqlgDataSourceFactory.java

```diff
@@ -0,0 +1,20 @@
+package org.umlg.sqlg.structure;
+
+import org.apache.commons.configuration.Configuration;
+import javax.sql.DataSource;
+
+/**
+ * Created by petercipov on 27/02/2017.
+ */
+public interface SqlgDataSourceFactory {
+
+    SqlgDataSource setup(String driver, final Configuration configuration) throws Exception;
+    SqlgDataSource setupFromJndi(String jndiName, Configuration configuration) throws Exception;
```

Comment:
Thanks, SqlgDataSource holds a method getPoolStatsAsJson that is also public. Can I remove it ?